src/midi/midi_mt32.cpp

```diff
@@ -0,0 +1,516 @@
+	        "Valid for rendering in separate thread only.");
+
+	int_prop = sec_prop.Add_int("partials", when_idle, 32);
+	int_prop->SetMinMax(8, 256);
+	int_prop->Set_help(
+	        "The maximum number of partials playing simultaneously. (min 8, max 256");
+
+	const char *mt32DACModes[] = {"0", "1", "2", "3", 0};
+	int_prop = sec_prop.Add_int("dac", when_idle, 0);
+	int_prop->Set_values(mt32DACModes);
+	int_prop->Set_help(
+	        "MT-32 DAC input emulation mode\n"
+	        "Nice = 0 - default\n"
+	        "Produces samples at double the volume, without tricks.\n"
+	        "Higher quality than the real devices\n\n"
+
+	        "Pure = 1\n"
+	        "Produces samples that exactly match the bits output from the emulated LA32.\n"
+	        "Nicer overdrive characteristics than the DAC hacks (it simply clips samples within range)\n"
+	        "Much less likely to overdrive than any other mode.\n"
+	        "Half the volume of any of the other modes.\n"
+	        "Perfect for developers while debugging :)\n\n"
+
+	        "GENERATION1 = 2\n"
+	        "Re-orders the LA32 output bits as in early generation MT-32s (according to Wikipedia).\n"
+	        "Bit order at DAC (where each number represents the original LA32 output bit number, and XX means the bit is always low):\n"
+	        "15 13 12 11 10 09 08 07 06 05 04 03 02 01 00 XX\n\n"
+
+	        "GENERATION2 = 3\n"
+	        "Re-orders the LA32 output bits as in later generations (personally confirmed on my CM-32L - KG).\n"
+	        "Bit order at DAC (where each number represents the original LA32 output bit number):\n"
+	        "15 13 12 11 10 09 08 07 06 05 04 03 02 01 00 14");
+
+	const char *mt32analogModes[] = {"0", "1", "2", "3", 0};
+	int_prop = sec_prop.Add_int("analog", when_idle, 2);
+	int_prop->Set_values(mt32analogModes);
+	int_prop->Set_help(
+	        "MT-32 analogue output emulation mode\n"
+	        "Digital = 0\n"
+	        "Only digital path is emulated. The output samples correspond to the digital output signal appeared at the DAC entrance.\n"
+	        "Fastest mode.\n\n"
+
+	        "Coarse = 1\n"
+	        "Coarse emulation of LPF circuit. High frequencies are boosted, sample rate remains unchanged.\n"
+	        "A bit better sounding but also a bit slower.\n\n"
+
+	        "Accurate = 2 - default\n"
+	        "Finer emulation of LPF circuit. Output signal is upsampled to 48 kHz to allow emulation of audible mirror spectra above 16 kHz,\n"
+	        "which is passed through the LPF circuit without significant attenuation.\n"
+	        "Sounding is closer to the analog output from real hardware but also slower than the modes 0 and 1.\n\n"
+
+	        "Oversampled = 3\n"
+	        "Same as the default mode 2 but the output signal is 2x oversampled, i.e. the output sample rate is 96 kHz.\n"
+	        "Even slower than all the other modes but better retains highest frequencies while further resampled in DOSBox mixer.");
```

Comment:
One of these settings is appropriate for our users. Rest makes sense for Munt project, but not for us (I think).